frontend/src/queries/nodes/DataTable/DataTableExport.tsx

```diff
@@ -26,7 +26,7 @@ import { ExporterFormat } from '~/types'
 import { dataTableLogic, DataTableRow } from './dataTableLogic'
 
 // Sync with posthog/hogql/constants.py
-export const MAX_SELECT_RETURNED_ROWS = 50000
+export const MAX_SELECT_RETURNED_ROWS = 300000
```

Comment:
why do you set this to 300K?

Reply:
this is the CSV export limit but the var is using the old name. CSV will export up to 300k. Just changed the naming to reflect